cloud/gateway/Dockerfile

```
@@ -0,0 +1,4 @@
+FROM eclipse-temurin:17
+EXPOSE 8080
+ADD build/libs/*.jar app.jar
```

Hadolint (2.12.0)

[error] Use COPY instead of ADD for files and folders

(DL3020)